src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1378,7 +1378,8 @@ struct
             partitions max_jobs_count unbundled_job_count )
     in
     Logger.info logger ~module_:__MODULE__ ~location:__LOC__
```

Comment:
I think this should be in debug. It's not something a user cares too much about.